pie.py

```diff
@@ -28,6 +30,10 @@
 # function to execute a command - must emulate the subprocess call method and return an error code on failure
 CMD_FN=subprocess.call
 
+# environment constant specific imports
+if WINDOWS:
+    import locale

```

Comment:
I'd move these two imports directly into the `_get_sys_prefix` function - after line 591. Keeps the link between the imports and the code that uses them closer.

Reply:
Cool. I was going to, but noticed there were no other inline imports so wasn't sure that was a convention you wanted to adopt or not. I'll change this.

Reply:
@adamkerz These have been moved.